pandas-stubs/core/series.pyi

```diff
@@ -2650,6 +2876,14 @@ class Series(IndexOpsMixin[S1], NDFrame):
         ),
     ) -> Series[complex]: ...
     @overload
+    def __sub__(
+        self: Series[Timestamp], other: datetime | np.datetime64 | np_ndarray_dt
+    ) -> TimedeltaSeries: ...
+    @overload
+    def __sub__(
+        self: Series[Timestamp], other: Series[Timestamp]
+    ) -> Series[Timedelta]: ...
```

Comment:
combine as one overload returning `TimedeltaSeries` and then convert it to this in the other PR

Reply:
OK